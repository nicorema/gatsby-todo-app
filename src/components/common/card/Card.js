import React from 'react'
import { StyledCard } from './Card.styles'

const Card = ({ children }) => <StyledCard>{children}</StyledCard>

export default Card
