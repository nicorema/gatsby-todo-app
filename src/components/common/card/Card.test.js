import React from 'react'
import renderer from 'react-test-renderer'

import Card from './Card'

describe('Card', () => {
  const CardCmp = renderer.create(
    <Card>
      <p>Testing child</p>
    </Card>
  )

  it('renders correctly', () => {
    const snapShot = CardCmp.toJSON()
    expect(snapShot).toMatchSnapshot()
  })
})
