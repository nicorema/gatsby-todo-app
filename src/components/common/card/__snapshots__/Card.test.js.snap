// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<div
  className="css-fmx4z3"
>
  <p>
    Testing child
  </p>
</div>
`;
